### ISSUE TITLE
Path issues on win32 system

### ISSUE THREAD

**Issue by marionebl** [patternplate/patternplate#192]
There are some issues around the win32 vs posix path delimiters:

```
$ npm run build:lsg && concurrently "npm run build:lsg -- -w" "patternplate start"

> pkg@0.7.0 build:lsg **\pkg
> tsc --project src/lsg --outDir build/lsg/patterns

[0]
[0] > pkg@0.7.0 build:lsg **\pkg
[0] > tsc --project src/lsg --outDir build/lsg/patterns "-w"
[0]
[1] Module not found: Error: Can't resolve './buildlsgpatternuttondemo.js' in '**\pkg'
[1] Module not found: Error: Can't resolve './buildlsgpatternuttondemo.js' in '**\pkg'
[1] Module not found: Error: Can't resolve './buildlsgpatterns  ab-navigationdemo.js' in '**\pkg'
[1] Module not found: Error: Can't resolve './buildlsgpatterns  ab-navigationdemo.js' in '**\pkg'
[1] Module not found: Error: Can't resolve './buildlsgpatternscolorsdemo.js' in '**\pkg'
[1] Module not found: Error: Can't resolve './buildlsgpatternscolorsdemo.js' in '**\pkg'
[1] Module not found: Error: Can't resolve './buildlsgpatternscopydemo.js' in '**\pkg'
[...]
```

First reported by @TheReincarnator

**Comment by marionebl** [comment]
Issues around components are resolved, remaining:

* no docs are found 
* check `build`
* check `create`

**Comment by marionebl** [comment]
Docs are fixed via 011580277d7dd6c09fa304df4fa4dca93d63f43a

**Comment by marionebl** [comment]
Issues should be solved in version `2.1.5` and up.
